File: PRAK205-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>
#include <math.h>
#include <string.h>

int main() {
    double A, B, C, luas, keliling;
    char input[100];

    printf("Masukkan nilai A (dan B jika format mendatar): ");
    fgets(input, sizeof(input), stdin);

    if (strchr(input, ' ') == NULL) {  
        sscanf(input, "%lf", &A);
        printf("B= ");
        scanf("%lf", &B);
    } else {  
        sscanf(input, "%lf %lf", &A, &B);
    }

    C = sqrt(B*B - A*A);  

    luas = 0.5 * A * C;  
    keliling = A + B + C;  

    printf("C(sisi alas)?\n");
    printf("Alas = %.0lf cm\n", C);
    printf("Tinggi = %.0lf cm\n", A);
    printf("Keliling = %.0lf cm\n", keliling);
    printf("Luas = %.0lf cm^2\n", luas);

    return 0;
}

File: MODUL 2/PRAK205-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
import math

def main():
    input_str = input("Masukkan nilai A (dan B jika format mendatar): ")

    if ' ' in input_str:  
        A, B = map(float, input_str.split())
    else:  
        A = float(input_str)
        B = float(input("B= "))

    C = math.sqrt(B*B - A*A)

    luas = 0.5 * A * C
    keliling = A + B + C

    print("C(sisi alas)?")
    print(f"Alas = {C:.0f} cm")
    print(f"Tinggi = {A:.0f} cm")
    print(f"Keliling = {keliling:.0f} cm")
    print(f"Luas = {luas:.0f} cm^2")

if __name__ == "__main__":
    main()
